lib/events.js

```diff
@@ -1,18 +1,57 @@
+const nacl = require("tweetnacl");
```

Comment:
For consistency (backward compatibility?), prefer

```suggestion
var nacl = require('tweetnacl');
```

Reply:
Agree